PowerPointLabs/PowerPointLabs/FunctionalTestInterface.Impl/PowerPointLabsFeatures.cs

```diff
@@ -85,7 +53,7 @@ public void CropOutPadding()
         {
             UIThreadExecutor.Execute(() =>
             {
-                var control = new RibbonControl("CropOutPaddingButton");
+                var control = new RibbonControl(TextCollection.CropOutPaddingTag);

```

Comment:
Can make it consistent by using "Ribbon.OnAction(new RibbonControl(TextCollection.AddCaptionsTag));"
There's a few others too.